web_chatter_camera/static/src/components/chatter_topbar/chatter_topbar.esm.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+import {ChatterTopbar} from "@mail/components/chatter_topbar/chatter_topbar";
+import {isMobileOS} from "@web/core/browser/feature_detection";
+import {patch} from "@web/core/utils/patch";
+import {useRef} from "@odoo/owl";
+
+const components = {ChatterTopbar};
+
+patch(
+    components.ChatterTopbar.prototype,
+    "mail_activity_board/static/src/components/chatter_topbar/chatter_topbar.esm.js",
```

Comment:
Wouldn't it be necessary to add this module to dependencies? --> [mail_activity_board](https://github.com/OCA/social/tree/17.0/mail_activity_board)